invenio_communities/assets/semantic-ui/js/invenio_communities/administration/components/RecordResourceActions.js

```diff
@@ -122,19 +123,19 @@ export class RecordResourceActions extends Component {
                 labelPosition="left"
               >
                 <Icon name="undo" />
-                {actionConfig.text}
+                {actionConfig.text}...
               </Element>
             );
           } else if (!resource.deletion_status.is_deleted && actionKey !== "restore") {
             return (
               <Element
                 key={actionKey}
-                onClick={this.onModalTriggerClick}
+                onClick={(e, data) => this.onModalTriggerClick(e, { ...data, actionConfig: actionConfig })}
```

Comment:
`Feature` button was braking the UI, because `actionConfig` had to be passed in line `ActionForm` in line 66